concept-anchor.md

```diff
@@ -0,0 +1,56 @@
+* Starting to explain InnerSource may be difficult unless one starts with a point decision makers are familiar with.
+* Agile/DevOps/Lean have already been heavily marketed and are commonly discussed terms in corporations, conferences, magazines etc.
+
+## Solutions
+### Agile
+There are several approaches that can be taken in the context of Agile depending on who's listening. The following main concepts are from [Wikipedia](https://en.wikipedia.org/wiki/Agile_software_development)
+* *Incremental and evolutionary processes* - Submitting pull requests for early feedback, or creating an issue for early stakeholder discussion aids this process, allowing ideas to be shaped and defined in the very early conceptual stages, thus being able to nourish good ideas and filter out unfeasible ideas before there are any significant costs associated with them.
+* *Efficient and face-to-face communication* - Many enterprises still implement collaboration models centered around face to face collaboration with teams being colocated. It is possible that management may view the "If it isn't written down and announced on the mailing list, it didn't happen" cultural norm of IS as unnecessary.
+However, one can highlight benefits to corporations from implementing IS practices.
+a) Business continuity in managing "Key man risk"
+When key people depart, the knowledge does not go out of the door with them. Robust documentation including decision records/RFCs and the like capture the rationale for important project design decisions and are key to continued productivity of new employees within the scrum team.
+b) Fostering productivity
+While the corporation's strategy may involve colocating employees who collaborate with each other regularly, it may still be a global corporation. Robust CI/CD and documentation facilitates collaboration and productivity across time zones and geographical locations. It eliminates barriers between teams and the production environment.
+* *Quality Focus* this is a major point for IS. Because of the expected submission by a variety of teams and individuals, extra care is taken to make sure that the submitted code is screened to be production quality. Comprehensive documentation for submitters, test coverage, CI/CD, static code analysis and other techniques are often used to screen code from strangers in the open source world, these techniques have great benefits for internal teams.
+
+##### [SCRUM](https://en.wikipedia.org/wiki/Scrum_%28software_development%29)
+To shorten the example and make it more concrete for SCRUM, a popular Agile methodology, one could cite the following scenario:
+Team A depends on Team B's code. Team A has a pressing priority for delivery, Team B has their own priorities, none of which is Team A, what to do? Using InnerSource, Team A can put the required code changes in to their own backlog and submit it to Team B, essentially removing block making things more "Agile".
```

Comment:
Two points:
- The example presented here is one of the three classic examples that are also present [here](https://github.com/InnerSourceCommons/InnerSourceLearningPath/tree/master/introduction) I think it would be a good idea to use the terms introduced there to make this point more concise and varied.
- It might be wise to highlight that Team B is still in control of what they accept or not, however are also strongly suggested to mentor Team A to ensure their contribution is to their standards - otherwise one might run into "oh so this means everyone can commit everywhere" discussions that usually go nowhere.